Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "image"
-version = "0.25.1"
+version = "0.25.2"
```

Comment:
The PR description says bumping the version field was needed for semver-checks? Might be worth seeing if we can avoid that so an accidental `cargo publish` with the repository checked out doesn't result in a new version upload to crates.io until we're actually ready to release

Reply:
This is because the semver check saw a change that required a minor version bump, and would continue to complain about it until the version has been bumped accordingly. 

I guess the CI could be changed to look only for semver-major changes.

Reply:
There's no harm bumping by two minor versions in a single release, so I think the _risk_ is at least low. At least lower than missing some of these warnings, I imagine. Maybe we can have a look at our release workflow instead, should be simple to compare against the latest published tag and do the version bump conditionally. I think this diff could also serve as one signal for when a release should be done.

Reply:
The failure mode I'm afraid of is me trying to publish a different crate but accidentally running `cargo publish` in the wrong terminal window. That'll push whatever branch I happen to have checked out as the next `image` release even if it just includes some half-baked in progress changes.

Reply:
Created https://github.com/image-rs/image/pull/2277 which seems to fix it

Reply:
Maybe dev version of the crate could have `publish = false` set?

or add `./cargo/config.toml` that somehow breaks the registry configuration?

```
[registry]
default = "…"  
```